pom.xml

```diff
@@ -402,13 +402,13 @@
         <dependency>
             <groupId>net.bytebuddy</groupId>
             <artifactId>byte-buddy</artifactId>
-            <version>1.14.11</version>
+            <version>1.15.7</version>
```

Comment:
Latest is v1.15.10. Upgrade?

Reply:
1.15.7 is the one that Selenium 2.46.0 was tested with. Didn't want to introduce additional compatibility issues.